packages/gasket-redux/src/configure-make-store.js

```diff
@@ -44,6 +44,7 @@ export function prepareReducer(allReducers, rootReducer) {
  */
 export default function configureMakeStore(
   {
+    devToolsOptions = {},
```

Comment:
Could you document this new option in the README.md for `@gasket/redux`?

Reply:
@rrex-godaddy is this PR still desirable? If so, could you please address the two items so we can get it merged and published?